vendor/modules.txt

```diff
@@ -1,30 +1,30 @@
-# github.com/klauspost/compress v1.11.7
+# github.com/klauspost/compress v1.11.12
 ## explicit
 github.com/klauspost/compress/s2
-# github.com/minio/highwayhash v1.0.0
+# github.com/minio/highwayhash v1.0.1
 ## explicit
 github.com/minio/highwayhash
-# github.com/nats-io/jwt/v2 v2.0.0-20210208203759-ff814ca5f813
+# github.com/nats-io/jwt/v2 v2.0.1
```

Comment:
Might be my machine but maybe we are missing jwt v1 here (which jwt/v2 imports internally), I got this locally after running `go mod vendor`:

```diff
+# github.com/nats-io/jwt v1.2.2
+github.com/nats-io/jwt
+# github.com/nats-io/jwt/v2 v2.0.1
 ## explicit
 github.com/nats-io/jwt/v2
```

Reply:
I don't, but doing a `go mod tidy` does update the go.sum...

Reply:
```